### PR TITLE
fix: add expire string parse.

### PR DESCRIPTION
**Pull request by Bidaya0**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryq.dev/en/main/contributing.html).

## Description

try to fix https://github.com/celery/celery/issues/8128.

### CODE REVIEW

**Review by auvipy — CHANGES_REQUESTED** [review]
and also check build failures for existing tests

File: celery/app/base.py [hunk 1]
```diff
@@ -738,6 +738,9 @@ def send_task(self, name, args=None, kwargs=None, countdown=None,
             if isinstance(expires, datetime):
                 expires_s = (maybe_make_aware(
                     expires) - self.now()).total_seconds()
+            if isinstance(expires, str):
```
[the remark this answers is not in the document]
Comment (Bidaya0) [reply celery/app/base.py]: > please also validate the change with tests

Tests have been added in t/unit/app/test_app.py

**Comment by auvipy** [comment]
thank you

**Comment by mililic85** [comment]
Hi, I have encountered the same error while using Celery primitives on my code.
Can you please advise me on how to check the Celery version in which this merge was impacted? I would like to see if this merge fixes my code. 
Celery used for me is version 5.2.7 in Python 10.5. 
Thanks a lot.

**Comment by auvipy** [comment]
you can see it is in 5.3.0 release milestone, you can try celery==5.3.0

**Pull request merged**